### PR TITLE
chore(README,actions.yaml): Improve documentation of the audience parameter

### LINKED ISSUE #13: OIDC Authentication Failure: Received Forbidden Response from JWT API
### Description:
I've been using the actions-oidc-debugger successfully in the past, but recently, without any changes to my configuration, it has stopped working and now returns a `Forbidden` error when attempting to authenticate using OIDC.

### Error Message:
```
received non-200 from jwt api: Forbidden
```

### Configuration:
Here's the relevant part of the GitHub Actions workflow configuration, which is closely based on the official documentation:

```yaml
jobs:
  debug-oidc:
    permissions:
      contents: read
      id-token: write
    runs-on: [self-hosted, micro]
    steps:
      - name: Checkout actions-oidc-debugger
        uses: actions/checkout@v3
        with:
          repository: github/actions-oidc-debugger
          ref: main
          path: ./.github/actions/actions-oidc-debugger
      - name: Debug OIDC Claims
        uses: ./.github/actions/actions-oidc-debugger
        with:
          audience: 'https://github.com/github'
```

### Expected Behavior:
The OIDC authentication should complete successfully as it had in the past, allowing the workflow to proceed without authentication errors.

### Actual Behavior:
The workflow fails during the OIDC authentication step with a Forbidden error, indicating a lack of permission or other authorization issues.

### Additional Context:
No changes were made to the GitHub Actions permissions or runner configuration prior to the onset of this issue.
This issue started occurring recently, suggesting a possible external change in the handling of OIDC requests or a change in API behavior.

Can you validate it still works the same on your side ?

### PR DESCRIPTION
**Pull request by 13013SwagR**
# Description
The current documentation is a bit unclear as to what is expected for the `audience`.
This change clarifies it.

## Related Issue
Fixes #13

### CODE REVIEW

**Review by nobe4 — APPROVED** [review]

File: action.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: 'OIDC Debugger'
 description: 'Print the GitHub Actions OIDC claims.'
 inputs:
   audience:
-    description: 'The audience to use when requesting the JWT.'
+    description: 'The audience to use when requesting the JWT. Your Github server URL and repository owner.'
```
Comment (nobe4) [inline action.yml:5]:
```suggestion
    description: 'The audience to use when requesting the JWT. Your Github server URL and repository owner (e.g. 'https://github.com/github').'
```

**Comment by dsotirakis** [comment]
woops, just saw that this exists. created https://github.com/github/actions-oidc-debugger/pull/15 myself, but going to close since you cover this case anyway. thanks for this, was also confused!

**Comment by 13013SwagR** [comment]
@nobe4 just curious about when you expect to run the workflows?

**Comment by nobe4** [comment]
Thanks for this @13013SwagR! I was off for a couple of day, and just came back :)

**Pull request merged**